N24DECE089/buoi5/cau2.cpp

```diff
@@ -0,0 +1,14 @@
+#include <iostream> 
+using namespace std ; 
+int main() 
+{ 
+    int n , i , k ; 
+    cout << " Moi nhap n " ; cin >> n ;
+    cout << " Day la bang cuu chuong " << n << endl ; 
+    for ( int i = 1 ; i <= n ; ++i ) 
+    {
+        for ( int k = 1 ; k < 10 ; ++k ) { cout << i*k << " " ;} 
+        cout << endl ; 
+    }
+    return 0 ; 
+}
```

Comment:
in ra bảng cửu chương. Dạng như n = 5 thì output sẽ là:
5x1=5
5x2=10
5x3=15
5x4=20
5x5=25
5x6=30
...

Reply:
Tại sao phải sử dụng tận 2 for lồng?